src/github.com/ebay/beam/rpc/kgobject.go

```diff
@@ -168,7 +169,7 @@ type WriteOpts struct {
 // the 'opts' can be used to control exactly what is written
 func (o KGObject) WriteTo(buff *bytes.Buffer, opts WriteOpts) {
 	if opts.NoLangID && o.IsType(KtString) {
-		buff.WriteString(o.value[0 : len(o.value)-19])
+		buff.WriteString(o.value[0 : len(o.value)-20])
```

Comment:
Please update the comment for `NoLangID` on line 164 to explain whether this includes or excludes the null terminator.